modules/ROOT/pages/endpoint-sftp-send.adoc

```diff
@@ -27,27 +38,27 @@ Send application messages (JSON or XML) to your backend applications in inbound
 |Yes
 
 |Username
-|Username for the SFTP Server.
+|The username for the SFTP Server.
 |Yes
 
 |Password
-|Password for the SFTP Server.
+|The password for the SFTP Server.
 |Yes
 
 |Target Path
-a|Path to the SFTP server EDI files:
+a|The path to the SFTP server EDI files:
 
 * That are generated from outbound message flows
 * Need to be placed for your trading partners
 * Where your backend applications require you to place translated JSON and XML files generated from inbound message flows
 |Yes
 
 |Filename pattern
-|xref:file-name-pattern.adoc[File name pattern] as required by your trading partner or backend application
+|The xref:file-name-pattern.adoc[file name pattern] required by your trading partner or backend application
```

Comment:
Note periods used above in descriptions; standardize per column per table
GLOBAL

Reply:
Done